ov7670/jpeg_encoder/src/runtime.c

```diff
@@ -117,7 +121,11 @@ bool runtimeParseArgs(Runtime* _runtime, int _argc, char* const _argv[])
           case 7  : cfg->m_rcConfig.m_fifoInput  = optarg;					break;
           case 7+1: cfg->m_rcConfig.m_fifoOutput = optarg;					break;
           case 7+2: cfg->m_rcConfig.m_videoOutEnable = atoi(optarg); break;
-
+          case 7+3: _runtime->m_state.m_targetJpgParams.jpgQuality = atoi(optarg); break;
+          case 7+4: 
+            if (strcmp(optarg, "false") == 0) _runtime->m_state.m_targetJpgParams.ifBlackAndWhite = false;
```

Comment:
Incorrect arg check.
There is already 0 in data structure (false).
Better to use `if (strcasecmp(optarg,"true")) _runtime->m_state.m_targetJpgParams.ifBlackAndWhite = false` otherwise any garbage will be treated as `true`